capstone/src/main/java/com/google/sps/servlets/DetailsForSearchResultsServlet.java

```diff
@@ -14,19 +14,41 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import com.google.maps.model.PlaceDetails;
+import com.google.sps.data.RestaurantDetailsGetter;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/get-details")
```

Comment:
Can we add a comment explaining what this servlet does? It fetches place details from the datastore when a user expands the restaurant info, right?